server/config.ts

```diff
@@ -34,7 +34,7 @@ export interface ApiConfig {
 }
 
 const auditConfig = () => {
-  const auditEnabled = toBoolean(get('AUDIT_ENABLED', 'false'))
+  const auditEnabled = toBoolean(get('AUDIT_ENABLED', false))
```

Comment:
Not related to this PR, but this was stopping the cypress tests running locally for me
More specifically it was stopping the server starting with `npm run start-feature` because `AUDIT_ENABLED` is not set in `feature.env`

Because `AUDIT_ENABLED` is not set in `feature.env`, it uses the fallback value which before my change was a string. 
The string value `'false'` doesn't work as intended in the `toBoolean` method as it's seen as a truthy value! (Man, I hate js's truthy and falsy values !)
Because it was seen as `true` the app tried to start with auditing enabled, but the rest of the required env vars are not set!

The fix is to simply pass this default value in as a `boolean` value 👍

Reply:
That's weird. It doesn't seem to do that for me when running locally. 🤔 If it fixes it though that's fine with me 🤷‍♂️